src/style.css

```diff
@@ -1 +1,12 @@
 /* styles go here */
```

Comment:
Unnecessary comments should be removed

Reply:
Ok, thanks, got it! I will take into account for the future.
Should I correct this task and re-deploy? Or can you leave it as is?

Reply:
done!)
the row with comments removed